pkg/assetfetcher/postgres.go

```diff
@@ -15,7 +15,7 @@ const fetchByIPQuery = `SELECT host(i.ip) as ip, c.resource_owner as resource_ow
 					  	RIGHT OUTER JOIN subnets s ON
```

Comment:
question: what's the rationale behind having a right outer join here? i.e. why do we need all subnets rather than the row we are interested in joining on?

cc @faldridge

Reply:
Good callout @zlozano. In the same vein, I'm not actually clear why line 18 needs to change from an inner join to a left outer join.

Reply:
Wait, I think I do understand this, nevermind.

Reply:
The left outer join is advantageous here since not every subnet will have a customer. So if that happens, we still want to return the left side of that join. If this were an inner join, and there was no customer_id, then the query would return 0 rows which may not be true. In the image attached, i give an example of this. The top result is from an inner join, while the bottom is resulting from a left outer join.

![image](https://user-images.githubusercontent.com/3370907/62737345-30e5bf80-b9f5-11e9-8fe8-f4f18bb02b93.png)

Reply:
Yeah, but you make a good point about the unnecessary RIGHT OUTER JOIN.  I'll revise it in a subsequent PR.